doc/test_in_no_std.md

```diff
@@ -0,0 +1,100 @@
+### test_in_no_std
+
+
+
+#### Run test in no_std environment.
+
+##### Requirements
+
+You need a nightly [Rust](https://www.rust-lang.org/) compiler with the `llvm-tools-preview` component, which can be installed through `rustup component add llvm-tools-preview` and `cargo install cargo-xbuild`.
+
```

Comment:
Maybe just make this into a code block to make it faster to "parse":

```bash
$ rustup install nightly
$ rustup component add llvm-tools-preview
$ cargo install cargo-xbuild
```

Reply:
Will add